packages/flutter/lib/src/widgets/scroll_configuration.dart

```diff
@@ -0,0 +1,37 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'basic.dart';
+import 'framework.dart';
+import 'scroll_behavior.dart';
+
+typedef Widget WrapScrollWidget(Widget scrollWidget);
```

Comment:
Never-mind. Since I'm transforming (ha) ScrollConfiguration into a delegate container, there's no need for this typedef.